test/Erc20Quest.spec.ts

```diff
@@ -307,7 +320,7 @@ describe('Erc20Quest', async () => {
       await ethers.provider.send('evm_increaseTime', [86400])
 
       const startingBalance = await deployedSampleErc20Contract.balanceOf(owner.address)
-      expect(startingBalance.toString()).to.equal('0')
+      expect(startingBalance).to.equal(0)
```

Comment:
Don't the values from the contract queries come back as BigNumber?

Reply:
Awesome, ran the tests and logged these and this seems to work for some reason lol. If no need to convert lets keep it simple!